dctest/upgrade_test.go

```diff
@@ -217,6 +222,9 @@ func testUpgrade() {
 				case "quay.io/cybozu/pause":
 					// Skip to check version because newer pause image is loaded after reboot
 					break
+				default:
+					// probably this test code does not follow cke
+					return errors.New("cke uses unknown container image.")
```

Comment:
A period at the end of an error message would be warned.

```suggestion
					return errors.New("cke uses unknown container image")
```